thrift/linkattacher/autogenerated/include/LinkAttacherServer.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+// This is an automatically generated file.
+// It could get re-generated if the ALLOW_IDL_GENERATION flag is on.
+
+#ifndef YARP_THRIFT_GENERATOR_LinkAttacherServer
+#define YARP_THRIFT_GENERATOR_LinkAttacherServer
+
+#include <yarp/os/Wire.h>
+#include <yarp/os/idl/WireTypes.h>
+
+namespace GazeboYarpPlugins {
+  class LinkAttacherServer;
+}
+
+
+class GazeboYarpPlugins::LinkAttacherServer : public yarp::os::Wire {
+public:
+  LinkAttacherServer();
+  /**
+   * Enable/disables gravity for a model
+   * @param model_name name that identifies model in gazebo (that are already spawned in gazebo)
+   * @param enable 1 to enable gravity, 0 otherwise
+   * @return returns true or false on success failure
+   */
+  virtual bool enableGravity(const std::string& model_name, const bool enable);
```

Comment:
If this is an abstract class you can append `= 0` to the virtual methods.

Reply:
@diegoferigo This is autogenerated.